src/rendering/contexts/json/context.zig

```diff
@@ -54,7 +56,7 @@ pub fn ContextType(
             };
         }
 
-        pub fn get(self: Context, path: Element.Path, index: ?usize) PathResolutionType(Context) {
+        pub fn get(self: Context, _: *DataRender, path: Element.Path, index: ?usize) PathResolutionType(Context) {
```

Comment:
Here another shortcut. We have a useless parameter for JSON content. We can see direct repercussions in tests: we initialize several `*DataRender` objects with `undefined` value to make this function works. This is the same problem in the ffi context file.
We need to proceed like this for `native` context.
Maybe the logic behind needs to be improved.